javascript/stimulus_reflex.js

```diff
@@ -532,7 +538,7 @@ if (!document.stimulusReflexInitialized) {
     const promise = reflexes[reflexId].promise
     const subjects = { error: true, halted: true, nothing: true, success: true }
 
-    if (element && subject == 'error') element.reflexError = body
+    if (element && subject == 'error') element.reflexError[reflexId] = body
```

Comment:
We should probably update this to use `===` [Strict Equality Check](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Strict_equality)